std/container/rbtree.d

```diff
@@ -1368,6 +1399,18 @@ assert(equal(rbt[], [5]));
         return removeKey(toRemove[]);
     }
 
+    ///
+    unittest
```

Comment:
It is both a boon and a curse. When I first wrote unit tests in here, I was able to do complete tests on all integer and string types, finding several bugs in the compiler in the process. But I soon learned that it screwed up user code that was unit testing. This bizarre scheme was the result. I hope we can fix the situation some day.